index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      media="screen"
+      href="css/index.css"
+    />
+    <script src="main.js"></script>
+  </head>
+  <html>
+    <body>
+      <main id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <figure id="img-div">
+          <img
+            id="image"
+            src="./image/tribute-page-main-image.jpg"
+            alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+          />
+          <figcaption id="img-caption">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+            on how to increase wheat yields - part of his life-long war on
+            hunger.
+          </figcaption>
```

Comment:
Good job on the html semantics but id love to recommend you avoid using so many id and use class instead. Check the following resource: https://www.educative.io/answers/class-and-id-selectors-in-css

Reply:
- You have done a great job using polite comments to pass your suggestions. 
- I'd like to also suggest that you make your comments sound a bit more friendly by adding emojis to them. 🤗 

_Kindly check the rest of my suggestions in my review summary below._ 🌸